bgs/bgs.go

```diff
@@ -116,10 +124,13 @@ func (bgs *BGS) HandleHealthCheck(c echo.Context) error {
 }
 
 type User struct {
-	gorm.Model
-	Handle string `gorm:"uniqueIndex"`
-	Did    string `gorm:"uniqueIndex"`
-	PDS    uint
+	ID        bsutil.Uid `gorm:"primarykey"`
+	CreatedAt time.Time
+	UpdatedAt time.Time
+	DeletedAt gorm.DeletedAt `gorm:"index"`
+	Handle    string         `gorm:"uniqueIndex"`
+	Did       string         `gorm:"uniqueIndex"`
+	PDS       uint
```

Comment:
Will the gorm auto-migrate code handle this sort of non-trivial migration? Or will we need to wipe state in staging to deploy this (`DROP DATABASE` and rebuild)?

As we move to having stuff deployed we should both add "deployment concerns" sections or comments to PR messages, so it is clear how and when they can go out.

Reply:
yeah this is actually not even a migration, its all the same fields, just one of them is getting a named type instead of a bare `uint`

Reply:
The `ID` field is new, and the primary key; will it be auto-generated on existing rows in the database?

Also all the timestamp fields are new, and don't seem to be nullable. Will they default to "now"?